src/main/java/primitiveTypes.java

```diff
@@ -1,2 +1,22 @@
 public class primitiveTypes {
```

Comment:
Press command+option+L on MAC or using menu Code -> Reformat code. Use this shortcut all time in all classes with which you work